nano/templates/shipsensors.tmpl

```diff
@@ -119,6 +119,31 @@
 	</table>
 {{/if}}
 </div>
+
+{{if data.id_name == 'Horizon'}}
+<div style="float:left;width:25%">
+<fieldset style="min-height:180px;background-color: #202020;">
+	<legend style="text-align:center">Inbound fire</legend>
+	<div class='item'>
+		<div class='item'>
+			{{:helper.link('', 'triangle-1-nw', { 'inbound_fire' : 'Fore-Port' }, null, null)}}
+			{{:helper.link('', 'triangle-1-n', { 'inbound_fire' : 'Fore' }, null, null)}}
+			{{:helper.link('', 'triangle-1-ne', { 'inbound_fire' : 'Fore-Starboard' }, null, null)}}
+		</div>
+		<div class='item'>
+			{{:helper.link('', 'triangle-1-w', { 'inbound_fire' : 'Port' }, null, null)}}
+			{{:helper.link('', 'circle-close', { 'inbound_fire' : 'clear' }, null, null)}}
+			{{:helper.link('', 'triangle-1-e', { 'inbound_fire' : 'Starboard' }, null, null)}}
+		</div>
+		<div class='item'>
+			{{:helper.link('', 'triangle-1-sw', { 'inbound_fire' : 'Bow-Port' }, null, null)}}
+			{{:helper.link('', 'triangle-1-s', { 'inbound_fire' : 'Bow' }, null, null)}}
+			{{:helper.link('', 'triangle-1-se', { 'inbound_fire' : 'Bow-Starboard' }, null, null)}}
+		</div>
+		</fieldset>
+</div>
+{{/if}}
```

Comment:
```suggestion
{{if data.id_name == 'Horizon'}}
<h4>Announce Incoming Projectiles</h4>
<div class='block'>
	<div class='item'>
		<div class='item'>
			{{:helper.link('', 'triangle-1-nw', { 'inbound_fire' : 'fore port' }, null, null)}}
			{{:helper.link('', 'triangle-1-n', { 'inbound_fire' : 'fore' }, null, null)}}
			{{:helper.link('', 'triangle-1-ne', { 'inbound_fire' : 'fore starboard' }, null, null)}}
		</div>
		<div class='item'>
			{{:helper.link('', 'triangle-1-w', { 'inbound_fire' : 'port' }, null, null)}}
			{{:helper.link('', 'circle-close', { 'inbound_fire' : 'clear' }, null, null)}}
			{{:helper.link('', 'triangle-1-e', { 'inbound_fire' : 'starboard' }, null, null)}}
		</div>
		<div class='item'>
			{{:helper.link('', 'triangle-1-sw', { 'inbound_fire' : 'aft port' }, null, null)}}
			{{:helper.link('', 'triangle-1-s', { 'inbound_fire' : 'aft' }, null, null)}}
			{{:helper.link('', 'triangle-1-se', { 'inbound_fire' : 'aft starboard' }, null, null)}}
		</div>
	</div>
</div>
{{/if}}
```
looks like this:
![image](https://user-images.githubusercontent.com/99297919/208320549-8f603ee2-4c19-41f7-85f9-f4e14ff0f61f.png)

Reply:
implemented new style suggestion in commit [0e6bc19](https://github.com/Aurorastation/Aurora.3/pull/15117/commits/0e6bc19af614dd7a1c8b2d3103c744fd24f1279a)